osu.Game/Overlays/Wiki/Markdown/WikiMarkdownContainer.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using Markdig.Extensions.Yaml;
+using Markdig.Syntax;
+using Markdig.Syntax.Inlines;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Containers.Markdown;
+using osu.Game.Graphics.Containers.Markdown;
+
+namespace osu.Game.Overlays.Wiki.Markdown
+{
+    public class WikiMarkdownContainer : OsuMarkdownContainer
+    {
+        public string CurrentPath
+        {
+            set => Schedule(() => DocumentUrl += $"wiki/{value}");
```

Comment:
What's the purpose of the schedule here?

Also feels very dodgy that this is using `+=`. I think it should be changed to not have things fall over if it is accidentally called more than once.

Reply:
When I try without schedule, the `DocumentUrl` is still empty when `CurrentPath` is set.

Maybe related to https://github.com/ppy/osu/pull/12947#discussion_r639405516